pymeasure/instruments/srs/sr830.py

```diff
@@ -89,6 +89,36 @@ class SR830(Instrument):
         """ Reads the Y value in Volts. """
     )
 
+    lia_status = Instrument.measurement("LIAS?",
+        """ Reads the value of the lockin amplifier (LIA) status byte. Returns a binary string with
+            positions within the string corresponding to different status flags:
+            bit 0: Input/Amplifier overload
+            bit 1: Time constant filter overload
+            bit 2: Output overload
+            bit 3: Reference unlock
+            bit 4: Detection frequency range switched
+            bit 5: Time constant changed indirectly
+            bit 6: Data storage triggered
+            bit 7: unused
+            """,
+        get_process=lambda s: bin(int(s))[2:],
+    )
+    
+    err_status = Instrument.measurement("ERRS?",
+        """Reads the value of the lockin error (ERR) status byte. Returns a binary string with
+           positions within the string corresponding to different error flags:
+           bit 0: unused
+           bit 1: backup error
+           bit 2: RAM error
+           bit 3: unused
+           bit 4: ROM error
+           bit 5: GPIB error
+           bit 6: DSP error
+           bit 7: Math error
+           """,
+        get_process=lambda s: bin(int(s))[2:],
+    )
+
```

Comment:
In both cases you return a bit mask which identifies the error conditions.
When users see the mask, they need to go back in the manual to decode the mask.
Now, python has some more descriptive data type for such cases.
For example, you could try to use `IntFlag` or `Flag` as suggested by @bilderbuchi
sometime ago in a similar situation.
In such a way users can directly see the meaning of the flags.

Reply:
Good point. I had not used the IntFlag type before this. Just implemented this change.